packit_service/worker/whitelist.py

```diff
@@ -239,12 +241,15 @@ def check_and_report(self, event: Optional[Any], project: GitProject) -> bool:
             account_name = event.github_login
             if not account_name:
                 raise KeyError(f"Failed to get account_name from {type(event)}")
-            if not self.is_approved(account_name):
-                logger.error(f"User {account_name} is not approved on whitelist!")
+            namespace = event.base_repo_namespace
+            # FIXME:
+            #  Why check account_name when we whitelist namespace only (in whitelist.add_account())?
+            if not (self.is_approved(account_name) or self.is_approved(namespace)):
```

Comment:
oh right, the collaborator check is elsewhere

Reply:
so should I change it to
```python
if not self.is_approved(namespace):
```
?